stories/data_stories.feature

```diff
@@ -1,34 +1,34 @@
 Feature: Data management
-    Values measured by controller blocks are captured
-    Configuration is captured
-    Configuration changes are captured
-    Events are captured
-    User annotations are captured
+Values measured by controller blocks are captured
+Configuration is captured
+Configuration changes are captured
+Events are captured
+User annotations are captured
 
-Scenario: View process data
-    Given there is an active process
-    And the active process contains UI blocks with a hardware association
-    When I click the View Process Data button
-    Then I should see all data generated by the process UI blocks
+    Scenario: Display process data
```

Comment:
Views get live updates automatically, no clicking should be needed outside of navigating to a view.